tasks/condition/condition.js

```diff
@@ -23,6 +30,28 @@ let task2;
 const alpha = 3;
 const beta = 7;
 const operator = "add";
+switch (operator) {
+  case operator === "add":
+    task2 == alpha + beta;
+    console.log(task2);
```

Comment:
please do not forget to remove console.log's while doing the final commit, it is a good habit.